templates/quickstart-confluence-master-with-vpc.template.yaml

```diff
@@ -0,0 +1,837 @@
+        - QSS3Region: !If
+            - GovCloudCondition
+            - s3-us-gov-west-1
+            - s3
+      Parameters:
+        AssociatePublicIpAddress: !Ref 'AssociatePublicIpAddress'
+
+        # NOTE: This is set to default due to CF parameter limits.
+        AutologinCookieAge: ''
+
+        CatalinaOpts: !Ref 'CatalinaOpts'
+        CidrBlock: !Ref 'CidrBlock'
+        ClusterNodeInstanceType: !Ref 'ClusterNodeInstanceType'
+        ClusterNodeMax: !Ref 'ClusterNodeMax'
+        ClusterNodeMin: !Ref 'ClusterNodeMin'
+        ClusterNodeVolumeSize: !Ref 'ClusterNodeVolumeSize'
+        CollaborativeEditingMode: !Ref 'CollaborativeEditingMode'
+        ConfluenceDownloadUrl: !Ref 'ConfluenceDownloadUrl'
+        ConfluenceVersion: !Ref 'ConfluenceVersion'
+        CustomDnsName: !Ref 'CustomDnsName'
+        DBInstanceClass: !Ref 'DBInstanceClass'
+        DBIops: !Ref 'DBIops'
+        DBMasterUserPassword: !Ref 'DBMasterUserPassword'
+        DBMultiAZ: !Ref 'DBMultiAZ'
+        DBPassword: !Ref 'DBPassword'
+        DBPoolMaxSize: !Ref 'DBPoolMaxSize'
+        DBPoolMinSize: !Ref 'DBPoolMinSize'
+        DBTimeout: !Ref 'DBTimeout'
+        DBIdleTestPeriod: !Ref 'DBIdleTestPeriod'
+        DBMaxStatements: !Ref 'DBMaxStatements'
+        DBValidate: !Ref 'DBValidate'
+        DBPreferredTestQuery: !Ref 'DBPreferredTestQuery'
+        DBAcquireIncrement: !Ref 'DBAcquireIncrement'
+        DBStorage: !Ref 'DBStorage'
+        DBStorageEncrypted: !Ref 'DBStorageEncrypted'
+        DBStorageType: !Ref 'DBStorageType'
+        DeployEnvironment: !Ref 'DeployEnvironment'
+        HostedZone: !Ref 'HostedZone'
+        JvmHeapOverride: !Ref 'JvmHeapOverride'
+        JvmHeapOverrideSynchrony: !Ref 'JvmHeapOverrideSynchrony'
+        KeyName: !Ref 'KeyName'
+        LocalAnsibleGitRepo: !Ref 'LocalAnsibleGitRepo'
+        LocalAnsibleGitSshKeyName: !Ref 'LocalAnsibleGitSshKeyName'
+        MailEnabled: !Ref 'MailEnabled'
+        SubDomainName: !Ref 'SubDomainName'
+        SSLCertificateARN: !Ref 'SSLCertificateARN'
+        StartCollectd: !Ref 'StartCollectd'
+        SynchronyClusterNodeMax: !Ref 'SynchronyClusterNodeMax'
+        SynchronyClusterNodeMin: !Ref 'SynchronyClusterNodeMin'
+        SynchronyNodeInstanceType: !Ref 'SynchronyNodeInstanceType'
+        TomcatAcceptCount: !Ref 'TomcatAcceptCount'
+        TomcatConnectionTimeout: !Ref 'TomcatConnectionTimeout'
+        TomcatContextPath: !Ref 'TomcatContextPath'
+        TomcatDefaultConnectorPort: !Ref 'TomcatDefaultConnectorPort'
+        TomcatEnableLookups: !Ref 'TomcatEnableLookups'
+        TomcatMaxThreads: !Ref 'TomcatMaxThreads'
+        TomcatMinSpareThreads: !Ref 'TomcatMinSpareThreads'
+        TomcatProtocol: !Ref 'TomcatProtocol'
+        TomcatRedirectPort: !Ref 'TomcatRedirectPort'
+        TomcatScheme: !Ref 'TomcatScheme'
+
+Outputs:
```

Comment:
I think it is worth adding BastionIP from the VPC stack as an Output. It is required to connect to most of the infrastructure anyway.